eclib/Makefile

```diff
@@ -8,25 +8,32 @@ CHECKS   ?= jasmin
 # --------------------------------------------------------------------
 DESTDIR  ?=
 PREFIX   ?= /usr/local
-BINDIR   := $(PREFIX)/bin
 LIBDIR   := $(PREFIX)/lib
-SHRDIR   := $(PREFIX)/share
 INSTALL  ?= install
 
 # --------------------------------------------------------------------
-.PHONY: default usage check install uninstall
+.PHONY: default usage check install uninstall clean
 
 default: check
 
 usage:
-	@echo "Usage: make <target> where <target> in [check|install|uninstall]" >&2
+	@echo "Usage: make <target> where <target> in [check|clean|install|uninstall]" >&2
 
 check:
 	easycrypt runtest $(ECARGS) $(ECCONF) $(CHECKS)
 
-install:
-	$(INSTALL) -m 0755 -d $(DESTDIR)$(LIBDIR)/jasmin/easycrypt
-	$(INSTALL) -m 0644 -t $(DESTDIR)$(LIBDIR)/jasmin/easycrypt *.ec
+jasmin.conf:
+	echo "[general]" | tee $@
+	echo "idirs=Jasmin:$(LIBDIR)/jasmin/easycrypt" | tee -a $@
```

Comment:
You swapped `jasmin` and `easycrypt` in the paths of `install:`, but not here. And there is `DESTDIR` in `install:` and not here. Are these differences intended?